src/app/workflow/workflow.component.html

```diff
@@ -346,7 +347,11 @@ <h3>
       </div>
     </div>
     <div fxLayout="column" fxLayoutGap="1rem" class="p-0 col-md-2 col-lg-3 ng-scope" *ngIf="isWorkflowPublic && workflow">
-      <app-launch-third-party [workflow]="workflow" [selectedVersion]="selectedVersion"></app-launch-third-party>
+      <app-launch-third-party
+        *ngIf="entryType !== EntryType.Tool"
```

Comment:
certain launch-with doesn't work?

Reply:
Idk if it doesn't work, but we don't ever display the launch-with buttons for tools